src/Bridges/Objective/bridges/slack.jl

```diff
@@ -174,3 +174,76 @@ function MOI.get(
     g = MOI.Utilities.remove_variable(f, bridge.slack)
     return MOI.Utilities.convert_approx(G, g)
 end
+
+"""
+    struct SlackBridgePrimalDualStart <: MOI.AbstractModelAttribute end
+
+[`Bridges.Objective.SlackBridge`](@ref) introduces a new constraint into the
+problem. However, because it is not a constraint bridge, it cannot intercept
+calls to set [`ConstraintDualStart`](@ref) or [`ConstraintPrimalStart`](@ref).
+
+As a work-around, set this attribute to `true` to set the primal and dual
+start for the new constraint. This attribute must be set after
+[`VariablePrimalStart`](@ref).
+"""
+struct SlackBridgePrimalDualStart <: MOI.AbstractModelAttribute end
+
+function MOI.throw_set_error_fallback(
+    ::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    ::Bool,
+)
+    return # Silently ignore if the model does not support.
+end
+
+function MOI.supports(
+    ::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    ::Type{<:SlackBridge},
+)
+    return true
+end
+
+function MOI.set(
+    model::MOI.ModelLike,
+    ::SlackBridgePrimalDualStart,
+    b::SlackBridge{T},
+    value::Bool,
+) where {T}
+    @assert value
+    # ConstraintDual: if the objective function had a dual, it would be `-1` for
+    # the Lagrangian function to be the same.
+    if MOI.supports(model, MOI.ConstraintDualStart(), typeof(b.constraint))
+        MOI.set(model, MOI.ConstraintDualStart(), b.constraint, -one(T))
+    end
+    # ConstraintPrimal: we should set the slack of f(x) - y to be 0, and the
+    # start of y to be f(x).
+    if !MOI.supports(model, MOI.VariablePrimalStart(), MOI.VariableIndex) ||
+       !MOI.supports(model, MOI.ConstraintPrimalStart(), typeof(b.constraint))
+        return
+    end
+    MOI.set(model, MOI.VariablePrimalStart(), b.slack, zero(T))
+    f = MOI.get(model, MOI.ConstraintFunction(), b.constraint)
+    f_val = MOI.Utilities.eval_variables(f) do v
+        return MOI.get(model, MOI.VariablePrimalStart(), v)
+    end
+    f_val -= MOI.constant(MOI.get(model, MOI.ConstraintSet(), b.constraint))
+    MOI.set(model, MOI.VariablePrimalStart(), b.slack, f_val)
+    MOI.set(model, MOI.ConstraintPrimalStart(), b.constraint, zero(T))
+    return
+end
+
+function MOI.set(
+    b::MOI.Bridges.AbstractBridgeOptimizer,
+    attr::SlackBridgePrimalDualStart,
+    value,
+)
+    if MOI.Bridges.is_objective_bridged(b)
```

Comment:
This will only work if the `SlackBridge` is the first objective bridge of the chain but it's enough for now if you add a comment saying we would do as complicated as `ObjectiveFunctionValue`/`ObjectiveValue` but we simplify since we know the slack bridge is usually the first one of the chain.